Test.C

```diff
@@ -1,53 +1,161 @@
+#include <stdio.h>
+#include <stdlib.h>
+
 /*
 
-Hello
+defining number literals
 
 */
 
-int main(){ 
-
-    //initialize game 
+#define NAME_LENGTH 20
+#define DESCRIPTION_LENGTH 200
+#define NUM_EQUIPMENT 10
+#define NUM_ENCRYPTION 10
+#define NUM_ATTACK 5
+
```

Comment:
Lets add a constant for the maximum amount of rounds a game can go with this pull request since you are setting up the `rounds` array inside of it.
```suggestion
#define NUM_ROUNDS 5

```